tests/network/vmi_istio.go

```diff
@@ -125,6 +126,59 @@ var _ = SIGDescribe("[Serial] Istio", func() {
 			By("Waiting for VMI to be ready")
 			tests.WaitUntilVMIReady(vmi, console.LoginToCirros)
 		})
+		Describe("Live Migration", func() {
+			BeforeEach(func() {
+				tests.SkipIfMigrationIsNotPossible()
+			})
+			migrationCompleted := func(migration *v1.VirtualMachineInstanceMigration) (bool, error) {
+				migration, err := virtClient.VirtualMachineInstanceMigration(migration.Namespace).Get(migration.Name, &metav1.GetOptions{})
+				if err != nil {
+					return false, err
+				}
+				if migration.Status.Phase == v1.MigrationSucceeded {
+					return true, nil
+				}
+				return false, fmt.Errorf("migration is in phase %s", migration.Status.Phase)
+			}
+			allContainersCompleted := func(podName string) (bool, error) {
+				pod, err := virtClient.CoreV1().Pods(vmi.Namespace).Get(context.TODO(), podName, metav1.GetOptions{})
+				if err != nil {
+					return false, err
+				}
+
+				for _, containerStatus := range pod.Status.ContainerStatuses {
+					if containerStatus.State.Terminated == nil {
+						return false, fmt.Errorf("container %s is not terminated, state: %s", containerStatus.Name, containerStatus.State.String())
+					}
+				}
+				return true, nil
+			}
+			It("All containers should complete in source virt-launcher pod after migration", func() {
+				sourcePodName := tests.GetVmPodName(virtClient, vmi)
+				migration := tests.NewRandomMigration(vmi.Name, vmi.Namespace)
+				migration, err = virtClient.VirtualMachineInstanceMigration(migration.Namespace).Create(migration)
+				Expect(err).NotTo(HaveOccurred())
+				assert.XFail("https://github.com/kubevirt/kubevirt/issues/5657", func() {
+					Eventually(func() error {
+						completed, err := migrationCompleted(migration)
+						if err != nil {
+							return err
+						}
+						if !completed {
+							return fmt.Errorf("migration of vmi with Istio proxy should complete")
+						}
```

Comment:
I would also move this to the `BeforeEach`.

Reply:
Done